Makefile

```diff
@@ -125,3 +125,12 @@ clean-dependencies:
 	rm -rf js/libs
 
 clean: clean-js-build clean-dependencies
+
+test-unit:
+	xvfb-run -s "-screen 0 1280x1024x24" testem -l Chromium ci --timeout 60
+
+test-integration:
+	xvfb-run -s "-screen 0 1280x1024x24" nosetests -s -w ui_tests
+
+server:
+	python -m SimpleHTTPServer 8080
```

Comment:
This should run ui_tests/server.py instead, as it also provides an interactive selenium driver environment, and produces test data to play with.

This is useful for developing new integration tests